### PR TITLE
Field Component

### PR DESCRIPTION
**Pull request by mxdi9i7**

### CODE REVIEW

**Comment by z695245771** [comment]
image

image
no background color in old page

File: src/components/Button/index.scss [hunk 1]
```diff
@@ -127,6 +127,24 @@ a.#{$baseClass} {
...
+  }
+
+  &__small {
+    height: 30px;
+    font-size: 12px;
+    font-weight: 300;
+    padding: 0 $space-md;
+  }
+
+  &__mini {
```
[the remark this answers is not in the document]
Comment (mxdi9i7) [reply src/components/Button/index.scss:142]: normal size is the default

**Comment by z695245771** [comment]
image
There should only be 5 px margin after icon, but there is 5 more pixel after icon container here

**Comment by z695245771** [comment]
Screen Shot 2020-05-27 at 1 08 47 PM

**Comment by z695245771** [comment]
Screen Shot 2020-05-27 at 1 11 27 PM
Screen Shot 2020-05-27 at 1 11 04 PM

**Comment by z695245771** [comment]
Screen Shot 2020-05-27 at 1 16 56 PM
why this popup twice

**Comment by z695245771** [comment]
Screen Shot 2020-05-27 at 1 21 43 PM
cant put punctuation marks at all, like  comma

**Comment by z695245771** [comment]
Screen Shot 2020-05-27 at 1 25 39 PM

**Comment by z695245771** [comment]
Screen Shot 2020-05-27 at 1 28 50 PM

**Comment by z695245771** [comment]
autoresize doesnt work too

**Comment by mxdi9i7** [comment]
> autoresize doesnt work too

Autoresize is a TODO, not implemented in this PR

**Comment by z695245771** [comment]
conflict dude

**Pull request merged**